### DISCUSSION TITLE
壁纸设置除了默认壁纸其他都无法正常使用，单独访问api链接是正常的

### DISCUSSION THREAD

**Discussion by haystack666** [imsyy/home#14] [General]
难道是因为壁纸设置是一个未开发完全的项目？

**Comment by imsyy** [comment]
我这边没问题啊，你清除一下缓存试试

**Comment by imsyy** [comment]
本地访问也有问题吗

**Comment by haystack666** [comment]
本地文件里面的十个图片是正常使用的，我指的是除开默认壁纸，其他的无法正常在网页中显示出来，比如每日一图、随机风景、随机动漫。但是我查看release包里面的壁纸api，当我单独提出来访问的时候都是可用的。

**Comment by imsyy** [comment]
现在应该OK了，我改成刷新后生效了

**Reply by haystack666** [reply]
非常感谢
